README.md

```diff
@@ -86,6 +86,56 @@ While we have purposefully avoided introducing JavaScript frameworks to concentr
 
 You can run this documentation offline by using [Docsify](https://docsify.js.org/#/). Fork this repo, [install Docsify](https://docsify.js.org/#/quickstart) on your local machine, and then in the root folder of this repo, type `docsify serve`. The website will be served on port 3000 on your localhost: `localhost:3000`.
 
+
+
+
+### Fork this repository
+
+Fork this repository by clicking on the fork button on the top of this page.
+This will create a copy of this repository in your account.
+
+
+### Clone the repository
+
+Now clone the forked repository to your machine. Go to your GitHub account, open the forked repository, click on the code button and then click the _copy to clipboard_ icon.
+
+Open a terminal and run the following git command:
+
```

Comment:
Hi👋 **Harshsharma-11**,
I must say you did great work making guidelines for setting up locally. Season on top, I kindly would like to suggest adding some steps for optional. I will state the step to reproduce below.

### Description
First thing first in this repository, I see the template review option to check or review the project inside the browser. In case you didn't know I will provide the screenshot below. If you clicked on the `Use this template`  you will be choosing whether you can preview it inside the codespace or create another repo. I'm humbly suggesting that feature would be nice if you state it inside your steps for someone who doesn't know that particular feature.

### Screenshots
![opensourceCap1](https://github.com/microsoft/Web-Dev-For-Beginners/assets/108471064/65f0e1d6-deb0-48d5-a68e-9f53a2ae3866)


Cheers mate, Happy coding!😄